src/platform/forms-system/src/js/widgets/CheckboxWidget.jsx

```diff
@@ -15,6 +15,13 @@ export default function CheckboxWidget({
     <span className="form-required-span">(*Required)</span>
   ) : null;
   const widgetClasses = classNames('form-checkbox', options.widgetClassNames);
+
+  const { widgetProps } = options;
+
+  const getProps = val => ({
```

Comment:
I recommend being a little clearer with this name. Some options include: `getWidgetProps`, `getInputuProps`, `propsForCheckbox`.